var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -215,10 +246,15 @@ def test(self):
             )
 
         # copy the environment configuration files to non-cache directories
-        op = ["run_env.sh", python_platlib + "/gptune/."]
-        self.run_test("cp", options=op, work_dir=wd)
-        op = ["run_env.sh", self.install_test_root + "/."]
-        self.run_test("cp", options=op, work_dir=wd)
+        ops = [
+            ["run_env.sh", python_platlib + "/gptune/."],
+            ["run_env.sh", self.install_test_root + "/."],
+        ]
+        for op in ops:
+            with test_part(self, f"test_part_cp_{op[1]}", purpose=f"gptune cp {op[1]} test"):
+                with working_dir(wd):
```

Comment:
Since these are copies, it would be MUCH better (if at all possible) to perform all of the setup in a `post-install` phase method.  

This could be done by leveraging the approach used for saving off build-time files (see https://spack.readthedocs.io/en/latest/packaging_guide.html#saving-build-and-install-time-files).

Reply:
Addressing this problem should resolve #44192 .